Quiz

TJBIII opened the pull request:
Quiz ready for initial review

chortlehoort commented:
@TJBIII General comments before I go into the code review.

1. When I first load your page and start typing into the text input, nothing happens and lots of console errors appear. You may want to prevent the user from typing before a card is selected.
2. Good functionality on the save button by clearing out the form and deselecting all cards.

chortlehoort commented on eventHandling.js:
```diff
@@ -0,0 +1,73 @@
+    document.getElementById('output').addEventListener("click", function(event){
+
+      //handle whether user is clicking on car row or if they are clicking right on the border
+      if (event.target.className.indexOf('row car')){
```
Consider using `event.currentTarget.getElementsByClass("car")` to eliminate the need for the `if..then` statement here

chortlehoort commented:
@TJBIII  Only one comment on your code for review. All in all, great looking code. Your comprehension is exactly where is need to be.

This is good enough to merge, but consider my comments first.  

:+1:

TJBIII commented:
I now disable the input field on page load and enable it once the user has clicked on a car. I removed the `if..then` statement and changed that click event listener to make my code more straightforward. Before, I was having a problem if the user clicked directly on the border, but now I don't have to work around that issue. Thanks for the comments!
